src/modules/form.js

```diff
@@ -0,0 +1,95 @@
+import { creator, createModal } from './helpers';
+import { setValues, symbols } from './display';
+import {
+  getInputsValues, modifyInput, getWeather, filterData,
+} from './data';
+import { setBGImage, iconList } from './background';
+
+const radioButtons = [
+  ['kelvin', '01'], ['celsius', '02'], ['fahrenheit', '03'],
+];
+
+const units = {
+  '01': '',
+  '02': '&units=metric',
+  '03': '&units=imperial',
+};
+
+function createRadioBtn(radioContainer, radioButtons, i) {
```

Comment:
**Let's make use of the arrow function instead of using the `function` keyword**
![image](https://user-images.githubusercontent.com/31897434/92687892-3b7b3280-f334-11ea-860a-cf5e33ef18f8.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/